.github/workflows/ci.yml

```diff
@@ -8,44 +8,38 @@ env:
 
 jobs:
   checkers:
+    name: Run static checkers
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - name: Run clang-format style check (.c and .h)
       uses: jidicula/clang-format-action@master
 
-  build:
+  sanitizers:
+    name: Build with ${{ matrix.compiler }} and -fsanitizer=${{ matrix.sanitizer }}
     runs-on: ubuntu-latest
-
     strategy:
+      fail-fast: false
       matrix:
+        sanitizer: [thread, undefined, leak, address]
         compiler: [gcc, clang]
```

Comment:
Runs without sanitizer would be good to have as well.

Reply:
might be possible with a supported if-statement in `actions`, will try it out

Reply:
Corrected now, works fine with empty string aswell